docs/src/create-apps/runtime-operations.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Runtime operations
+description: Set up runtime operations to run one-off commands on your project through the Platform.sh API.
+weight: 6
+---
+
+Runtime operations allow you to trigger one-off commands or scripts on your project.
+Similar to [crons](../create-apps/app-reference.md#crons), they run in the app container but not on a specific schedule.
+You can [define runtime operations](#define-a-runtime-operation) in your [app configuration](../create-apps/app-reference.md)
+and [trigger them](#run-a-runtime-operation) at any time through the Platform.sh API using a cURL command.
+
+For example, if you have a static website,
+you may want to set up a runtime operation to occasionally fetch content from a backend system
+without having to rebuild your whole app.
+
+You can use runtime operations if you have Grid or {{% names/dedicated-gen-3 %}} environments.
+
+## Define a runtime operation
+
+To define a runtime operation, add a configuration similar to the following:
+
+```yaml {location=".platform.app.yaml"}
+operations:
+  {{< variable "RUNTIME_OPERATION_NAME" >}}:
+    role: {{< variable "USER_ROLE" >}}
+    commands:
+      start: {{< variable "COMMAND" >}}
+```
+
+When you define a runtime operation,
+you can specify which users can trigger it according to their user `role`:
+
+- `viewer`
+- `contributor`
+- `admin`
+
+If you don't set the `role` option when configuring your runtime operation,
+by default all users with the `contributor` role can trigger it. 
+
+For example, to allow admin users to clear the cache of a Drupal site,
+you could define an operation like the following:
+
+```yaml {location=".platform.app.yaml"}
+operations:
+  clear-rebuild:
+    role: admin
+    commands:
+      start: drush cache:rebuild
+```
+
+The name of the runtime operation in this case is `clear-rebuild`.
+
+For more possibilities, see other [runtime operation examples](#runtime-operation-examples). 
+
+## Run a runtime operation
+
+Once you've [defined a runtime operation](#define-a-runtime-operation), 
+you can trigger it through the Platform.sh API.
+To do so, run a cURL command similar to the following:
+
+```bash
+platform p:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/deployments/current/operations -X POST -d '{"operation": "{{< variable "RUNTIME_OPERATION_NAME" >}}", "service": "{{< variable "CONTAINER_NAME" >}}"}' -p {{< variable "PROJECT_ID" >}}
+```
+
+You can only trigger a runtime operation if you have permission to do so.
+Permissions are granted through the `role` option specified in the [runtime operation configuration](#define-a-runtime-operation).
+
+For example, to trigger the runtime operation [defined previously](#define-a-runtime-operation),
+you could use the following command:
+
+```bash
+$ platform p:curl /environments/environment_abcd1234/deployments/current/operations -X POST -d '{"operation": "clear-rebuild", "service": "app"}' -p abcdefgh1234567
```

Comment:
This command doesn't follow the same rules than the others one:
- it starts with a `$`
- it uses `environment_abcd1234` instead of `{{< variable "ENVIRONMENT_ID" >}}`
- it uses abcdefgh1234567  instead of `{{< variable "PROJECT_ID" >}}`